pkg/clients/command/mgmt.go

```diff
@@ -0,0 +1,87 @@
+package command
```

Comment:
I created PR #683 based in your work. If this PR is accepted, please discard this client and use the new management client.

Reply:
@jduranf Making a note, thanks.